CRM/CiviAwards/Upgrader/Steps/Step1005.php

```diff
@@ -37,7 +37,7 @@ public function apply() {
     foreach ($result['values'] as $id => $value) {
       civicrm_api3('OptionValue', 'create', [
         'id' => $id,
-        'description' => 'CRM_CiviAwards_Helper_CaseTypeCategory::getSubTypes;',
+        'description' => 'CRM_CiviAwards_Helper_CaseTypeCategory::getAwardSubtypes;',
```

Comment:
We don't need to change this. Will cause problems and the change in name is already accounted for in the function.